### ISSUE TITLE
enhance the introduction of the application with current information inthe database

### ISSUE THREAD

**Issue by Ebaneck** [Ebaneck/Epita-Java-Project#5]
Add a view option to list all what is contained within the database to the user upon launching the application. this gives light on the possible interactions and operation that the user can perform.

@Donabay I will need help from you on how to present this information to users.
This is my suggestion:
1. should we add just a select option to view all this information or what were you thinking of implementing for this enhancement.

**Comment by Ebaneck** [comment]
this is what i have done for this enhancement. i added a list option on the introductory screen such that the user can list and show all the information currently available within the database.








<img width="960" alt="hj" src="https://user-images.githubusercontent.com/10956602/36172073-12eae590-1105-11e8-827e-53c6631d4154.png">

**Comment by Donabay** [comment]
yes...this is what i wanted. i want to add another option which is search for an identity. The user might want to search for an identity before performing any action. What do u think ?

**Comment by Donabay** [comment]
Yeah this is what i wanted. I was thinking of adding a search option which will allow the user to search for a specific identify before performing any action on it. What do you think?
